erts/doc/src/erlang.xml

```diff
@@ -6347,6 +6353,14 @@ true</pre>
         <pre>
 <input>round(-5.5).</input>
 -6</pre>
+        <pre>
+<input>round(36028797018963969.0).</input>
```

Comment:
I changed the float literal that you suggested for another example were I wanted to show that a float is not always printed the same way as the float literal that the float was created from and then I thought that it would be pedagogically advantageous to use the same float literal in all examples.

Reply:
aha!